### DISCUSSION TITLE
Haiku

### DISCUSSION THREAD

**Discussion by baimafeima** [speed47/spectre-meltdown-checker#551] [General]
I've installed [Haiku](https://www.haiku-os.org/) over the weekend and was so impressed by the new beta that I'm considering to leave the Linux world for it. Out of curiosity - as I understand Haiku has its very own non-monolithic modular kernel - what would it take to make use of an application like `spectre-meltdown-checker` on Haiku. Would this even make any sense?

**Comment by lilyanatia** [comment]
it should be possible to determine whether the hardware is vulnerable or not, but it looks unlikely that mitigations for these vulnerabilities will ever be implemented in Haiku: https://discuss.haiku-os.org/t/intel-bug-security/6394
